content/docs/command-reference/exp/show.md

```diff
@@ -168,6 +168,11 @@ $ dvc exp show ...
   automatically. You can enable `dvc config plots.auto_open` to make this the
   default behavior.
 
+- `-f`, `--force` - force re-collection of experiments instead of loading from
+  exp cache. DVC caches `exp show` table data to improve performance of
```

Comment:
exp cache -> cache

Reply:
This is completely separate from DVC cache (and is not stored in `.dvc/cache`) so I was not sure if just using `cache` here in the docs would be confusing

Reply:
kk, makes sense ... can we do "the internal experiments cache" or at least "experiments cache"